processor/src/main/java/org/infinispan/protostream/annotations/impl/processor/ProtoSchemaAnnotation.java

```diff
@@ -20,15 +19,14 @@ class ProtoSchemaAnnotation {
    private final String schemaPackageName;
    private final boolean service;
    private final ProtoSyntax syntax;
-   private boolean allowNullFields;
+   private final boolean allowNullFields;
```

Comment:
seems unrelated. is it ok?

Reply:
Yeah, but it's such a tiny thing that I didn't want to create a different PR for that